config/maintainers.json

```
@@ -1,6 +1,16 @@
 {
   "docs_url": "https://github.com/exercism/docs/blob/master/maintaining-a-track/maintainer-configuration.md",
   "maintainers": [
+    {
+      "github_username": "rishiosaur",
+      "alumnus": true,
+      "show_on_website": true,
+      "name": "Rishi Kothari",
+      "link_text": "Website",
+      "link_url": "https://rishi.js.org",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/15793261?s=460&u=dce39daca78b2f794644abdf061eed9a425d2a2c&v=4",
```

```suggestion
      "avatar_url": null,
```

We pull your GH avatar! So no need to set it :D